includes/class-micropub-endpoint.php

```diff
@@ -49,8 +49,18 @@ public static function return_micropub_error( $response, $handler, $request ) {
 		return $response;
 	}
 
-
-
+	public static function log_error( $message, $name = '' ) {
```

Comment:
If you set  `'Micropub'` as default for `$name`, you do not have to check in line 59.